#std_msgs/Header header
float32 linear
float32 angular
float32 odometer_x
float32 odometer_y
float32 yaw_angle
int32 battery_voltage
int32 left_encoder
int32 right_encoder
int8  keystate
int8  bump
int16 left_obs
int16 centre_obs
int16 right_obs
int16 right_wall
int16 left_nec
int16 f_left_nec
int16 f_right_nec
int16 right_nec
int16 brush_current
int16 left_wheel_current
int16 right_wheel_current
int16 robot_current
int32 system_ticks
